code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -62,6 +62,87 @@
 	construction_time = 200
 	category = list("Cyborg")
 
+//Prosthetics
+/datum/design/prosthetic_l_arm
+	name = "Surplus Prosthetic Left Arm"
+	id = "prosthetic_l_arm"
+	build_type = PROTOLATHE | MECHFAB
+	build_path = /obj/item/bodypart/l_arm/robot/surplus
+	materials = list(/datum/material/iron=3000)
+	construction_time = 50
```

Comment:
use SECONDS defines

Reply:
SECONDS defines?

Reply:
`5 SECONDS` instead of `50`

Reply:
I will do this but would like to point out not a single other mechfab construction time uses that format.

Reply:
thats because they are old as fuck